source/index.rst

```diff
@@ -252,5 +255,6 @@ be interesting to you.
    :hidden:
 
    about
+   fitech_info
```

Comment:
```suggestion
   fitech-info
```
I suggest the file be named with a `-` instead of `_`, that's more typical of what we use.

Reply:
acknowlegdged